.gitattributes

```diff
@@ -0,0 +1,11 @@
+.gitattributes export-ignore
+.gitignore export-ignore
+
+.github export-ignore
+.editorconfig export-ignore
+
+# Dev & test files
+Makefile export-ignore
+/php_env export-ignore
+phpunit.xml export-ignore
```

Comment:
This one should be removed or the tests directory be added back
I missed that

Anyway I doubt anyone runs tests using the composer package